chrome/install.js

```diff
@@ -74,6 +74,12 @@ function onLoad() {
 				if(success) wizard.advance();
 			}
 		});
+	} else {
+		checkMacJDK().then(function() {
+			wizard.getPageById("intro").next = "openoffice-installations";
+		}).catch(function() {
+			wizard.getPageById("intro").next = "jdk-required";
+		})
```

Comment:
Resolve as true (openoffice-installation) or false (jdk-required), and use catch for any other problem.